extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/Basic.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.vertx.http.runtime.security;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Selects {@link BasicAuthenticationMechanism}.
+ * Equivalent to '@HttpAuthMechanism("basic")'.
+ *
+ * @see HttpAuthMechanism for more information
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE, ElementType.METHOD })
+public @interface Basic {
```

Comment:
```suggestion
public @interface BasicAuthentication {
```

Reply:
It appears that this and other annotations should be the extensions of `@HttpAuthenticationMechanism`, similarly to the JAX-RS `@GET` being the extension of `@HttpMethod`

Reply:
Done.